sdk/eventgrid/Azure.ResourceManager.EventGrid/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "net",
+  "TagPrefix": "net/eventgrid/Azure.ResourceManager.EventGrid",
```

Comment:
Please wait until https://github.com/Azure/azure-sdk-for-net/pull/37003 got merged, then update this pr and merge